src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -121,6 +121,13 @@ public boolean match(final Unit unit) {
       return (ua.getIsCombatTransport() && ua.getIsSea());
     }
   };
+  public static final Match<Unit> UnitIsDefenselessTransport = new Match<Unit>() {
```

Comment:
Thanks. As a suggestion to try and make you successful:
- when writing test, focus on testing exactly one thing, one piece of functionality.

For structure:
- create a new method in battle tracker that is then called once (this is the merged method)
- have that method then call three package protected methods. One of those methods will be the very first air combat method that we wrote earlier.
- Then finally test the three package protected methods on their own, following the same rough pattern we had for the first one.